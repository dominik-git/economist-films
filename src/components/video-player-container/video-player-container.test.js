// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import VideoPlayerContainer from './video-player-container';

test('renders correctly when videoUrl state is null', () => {
  const tree : string = renderer.create(<VideoPlayerContainer params={{ id: 50 }} />).toJSON();
  expect(tree).toMatchSnapshot();
});
test('renders correctly when videoUrl state is not null', () => {
  const app = mount(<VideoPlayerContainer params={{ id: 50 }} />);
  app.setState({ videoUrl: 'https://cdn-films.economist.com/DW/MAY01_REV/MTMYSCivil.m3u8' });
  expect(app.state().videoUrl).toEqual('https://cdn-films.economist.com/DW/MAY01_REV/MTMYSCivil.m3u8');
});
