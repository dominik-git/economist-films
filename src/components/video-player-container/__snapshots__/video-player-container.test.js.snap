// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when videoUrl state is null 1`] = `
<div
  className="video-player-container"
/>
`;
